client/my-sites/stats/stats-interval-dropdown/style.scss

```diff
@@ -0,0 +1,68 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@automattic/components/src/styles/typography.scss";
+
+
+.stats-interval-dropdown__container {
+
+	> .components-button {
+		border-radius: 2px;
+		color: var(--gutenberg-gray-900);
+		font-family: $font-sf-pro-text;
+		font-size: 13px; /* stylelint-disable-line declaration-property-unit-allowed-list */
+		font-weight: 400;
+	}
+
+	.segmented-control.is-primary.stats-navigation__intervals .segmented-control__item.is-selected .segmented-control__link:hover {
```

Comment:
Is it possible to shorten this chain (in the following PR)? This would help in case it needs to be overwritten in the future.

Reply:
Thanks @grzegorz-cp ! I will give it a shot in a followup PR and see. Specificity & overriding the styles from the dropdown component has been the main challenge of this so far, so I will see if we can shorten it in any way further.